addons/skin.estuary/xml/DialogGameControllers.xml

```diff
@@ -4,6 +4,7 @@
 	<include>Animation_DialogPopupOpenClose</include>
 	<controls>
 		<include condition="Window.IsActive(gamecontrollers)">GameDialogControllers</include>
+		<include condition="Window.IsActive(gameports)">GameDialogPorts</include>
 	</controls>
 	<controls>
```

Comment:
@garbear you have an extra group of `<control></controls>` here.

Reply:
Thanks, fixed: https://github.com/xbmc/xbmc/pull/22641